libs/features/mas/src/aem-fragment.js

```diff
@@ -213,7 +213,7 @@ export class AemFragment extends HTMLElement {
     }
 
     #transformAuthorData() {
-        const { fields, id, tags } = this.#rawData;
+        const { fields, id, tags, } = this.#rawData;
```

Comment:
Looks like the comma after `tags` in not needed

Reply:
removed